src/components/MatchesTable.vue

```diff
@@ -48,6 +48,32 @@
         {{ item.team2_string }}
       </div>
     </template>
+
+    <template v-slot:item.ip_string="{ item }">
+      <div>
+          {{item.display_name}}
+      </div>
+    </template>
+
+    <template v-slot:item.connect="{ item }">
```

Comment:
Maybe have a check only if the person is a super_admin/admin or is a part of the match to show the connect button. And maybe remove the button when the match is completed.

Reply:
I've add a v-if on the match.end_time to display the connect and gotv button.

Reply:
I'm fairly certain that is a configuration issue. Please revert this change as it was working prior to by having the `v-if` on the template.